packages/web/src/new/services/Repositories/WalletsRepository.ts

```diff
@@ -67,6 +68,12 @@ export class WalletsRepository extends SDListViewModel<Wallet> {
     this._stopObserving();
   }
 
+  override reload(): void {
+    if (!this._wasRequested) {
```

Comment:
reload never calls twice after because of this condition, and because `_wasRequested` never became false. do you think its right?